crates/notedeck/src/i18n/manager.rs

```diff
@@ -0,0 +1,766 @@
+use fluent::FluentArgs;
+use fluent::{FluentBundle, FluentResource};
+use fluent_langneg::negotiate_languages;
+use std::collections::HashMap;
+use std::path::Path;
+use std::sync::{Arc, RwLock};
+use unic_langid::LanguageIdentifier;
+
+/// Manages localization resources and provides localized strings
+pub struct LocalizationManager {
+    /// Current locale
+    current_locale: RwLock<LanguageIdentifier>,
+    /// Available locales
+    available_locales: Vec<LanguageIdentifier>,
+    /// Fallback locale
+    fallback_locale: LanguageIdentifier,
+    /// Resource directory path
+    resource_dir: std::path::PathBuf,
+    /// Cached parsed FluentResource per locale
+    resource_cache: RwLock<HashMap<LanguageIdentifier, Arc<FluentResource>>>,
+    /// Cached string results per locale (only for strings without arguments)
+    string_cache: RwLock<HashMap<LanguageIdentifier, HashMap<String, String>>>,
+}
+
+impl LocalizationManager {
+    /// Creates a new LocalizationManager with the specified resource directory
+    pub fn new(resource_dir: &Path) -> Result<Self, Box<dyn std::error::Error + Send + Sync>> {
+        // Default to English (US)
+        let default_locale: LanguageIdentifier = "en-US"
```

Comment:
For now, it's en-US, but later on, we'll change this to preferred locale in Notedeck settings OR system locale (if we support it).